tv_apps/smart-system/manifest.webapp

```diff
@@ -11,6 +11,7 @@
   "permissions": {
     "themeable":{},
     "alarms": {},
+    "system-app-only-audio-channels-in-app": {},
```

Comment:
Since we don't handle `mozChromeEvent` in this patch, should we need this permission?